### PR TITLE
feat(config): document ‹status_name_template›

### PR DESCRIPTION
**Pull request by mfocko**
Merge with packit/packit-service#2402

### CODE REVIEW

File: docs/configuration/index.md [hunk 1]
```diff
@@ -684,6 +684,25 @@ require:
 
 ```
 
+#### status_name_template
+
+(*string*)  Template that can be used to modify the (commit) status name that
+Packit uses by default. It used in the following manner:
```
Comment (nforro) [inline docs/configuration/index.md:690]:
```suggestion
Packit uses by default. It is used in the following manner:
```

**Review by nforro — APPROVED** [review]
Maybe we could link the [Format String Syntax](https://docs.python.org/3/library/string.html#formatstrings)?

**Review by lbarcziova — COMMENTED** [review]
we should probably warn that it is on the user to make sure there will be a unique status name for each job run, i.e. Packit doesn't overwrite the commit status because there are 2 job runs resulting in the same status name

**Review by lachmanfrantisek — APPROVED** [review]
Thanks!

File: docs/configuration/index.md [hunk 2]
```diff
@@ -684,6 +684,25 @@ require:
...
+```py
+status_name_template.format(…)
+```
+
+This allows for automation with GitHub merge queues, or any other tools that
+depend on the consistent status naming.
+
+We provide the following variables:
+- `job_name`, e.g., `rpm-build` or `testing-farm`
+- `chroot`
```
[the remark this answers is not in the document]
Comment (mfocko) [reply docs/configuration/index.md]: added

File: docs/configuration/index.md [hunk 3]
```diff
@@ -684,6 +684,25 @@ require:
...
+#### status_name_template
+
+(*string*)  Template that can be used to modify the (commit) status name that
+Packit uses by default. It used in the following manner:
+
+```py
+status_name_template.format(…)
+```
+
+This allows for automation with GitHub merge queues, or any other tools that
```
[the remark this answers is not in the document]
Comment (mfocko) [reply docs/configuration/index.md:715]: I'll add an example after some testing.

**Pull request merged**